packages/base-zone/test/prepare-test-env-ava.js

```diff
@@ -1,5 +1,4 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+import { wrapTest } from '@endo/ses-ava';
+import rawTest from 'ava';
 
-import test from 'ava';
-
-export { test };
+export const test = wrapTest(rawTest);
```

Comment:
This also changes the existing base-zone tests to use ses-ava rather than ava. Unless there was a specific reason not to, this is an improvement in general. Though it was motivated to preserve this feature of test-prepare-revocable.js, its effect on the other local tests here seems to me to be harmless and beneficial. Am I missing anything?

Reply:
I don't know the reason folks started to move away from `@endo/ses-ava`.  Maybe @turadg knows the details?

Reply:
I've used regular Ava when I run into a problem using ses-ava:
- https://github.com/endojs/endo/issues/611
- https://github.com/endojs/endo/issues/647
- https://github.com/endojs/endo/issues/891
- https://github.com/endojs/endo/issues/1467